Dockerfile

```diff
@@ -1,31 +1,11 @@
-FROM python:3.7.6-alpine3.10
-
-RUN apk add --no-cache \
-        bash \
-        ca-certificates \
-        g++ \
-        gcc \
-        git \
-        libressl \
-        libressl-dev \
-        linux-headers \
-        make \
-        musl-dev \
-        zlib-dev
-
-ARG LIBRDKAFKA_VERSION=1.0.1
-RUN wget https://github.com/edenhill/librdkafka/archive/v$LIBRDKAFKA_VERSION.tar.gz \
-        -O /tmp/librdkafka-$LIBRDKAFKA_VERSION.tar.gz && \
-    cd /tmp/ && \
-    tar zxf librdkafka-$LIBRDKAFKA_VERSION.tar.gz && \
-    cd librdkafka-$LIBRDKAFKA_VERSION && \
-    ./configure && \
-    make && \
-    make install && \
-    cd .. && rm -fr librdkafka-$LIBRDKAFKA_VERSION
-
+FROM python:3.8-slim-buster
```

Comment:
We need to verify that 3.8 works for our services. I know I had trouble with testing with 3.8. I can reproduce if necessary.

Reply:
@simonlindroosfyndiq For sure! I have only tested with the `order-handler-service` and it seems to be working

Reply:
The issue we were experiencing was that 3.8 did not work with an old version of `structlog`. Upgrading `structlog` and 3.8 seems to work.